utensor_cgen/templates/snippets/argmax_op.cpp

```diff
@@ -1,6 +1,19 @@
+{% if create_sptr %}
```

Comment:
There is a lot of code reuse in these blocks. Let's make them another template and import it here

Reply:
Yay, you are right.
I just didn't have time to figure out how to organize where to put the import template files.
So I just copy&paste for now.

Reply:
Cool I'll take a look at this